webpack/assets/javascripts/react_app/components/Architectures/Welcome.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { translate as __ } from '../../common/I18n';
+import EmptyState from '../common/EmptyState';
+import { foremanUrl } from '../../common/helpers';
+
+export const WelcomeArchitecture = ({ canCreate }) => {
+  const action = canCreate && {
+    title: __('Create Architecture'),
+    url: foremanUrl('/architectures/new'),
+  };
+  const description = (
+    <>
+      <p>
+        {__(
+          'Before you proceed to using Foreman you should provide information about one or more architectures.'
+        )}
+        <br />
+        {__(
+          'Each entries represent a particular hardware architecture, most commonly'
+        )}
+        <b>{__(' x86_64')}</b> {__('or')} <b>{__('i386.')}</b> <br />
+        Foreman also supports the Solaris operating system family, which
```

Comment:
You will need to add a couple of strings to https://github.com/theforeman/foreman/blob/develop/.eslintrc#L13 so the spell checker doesn't complain.
Also, this string is not marked for translation.